packages/keycloak/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.28.1"
```

```suggestion
- version: "1.29.0"
```
(enhancements bump minor)